packages/components/src/date-time/date/index.tsx

```diff
@@ -1,7 +1,10 @@
 /**
  * External dependencies
  */
-import { useLilius } from 'use-lilius';
+/**
+ * Internal dependencies
+ */
+import { useLilius } from './use-lilius';
```

Comment:
Nit: import statements seem to be misplaced 😅

![import-statement](https://github.com/user-attachments/assets/dc6cbd1d-40d6-43d1-b0eb-0c5261db1287)

Reply:
Well spotted, I fixed it in https://github.com/WordPress/gutenberg/pull/65097/commits/903ab49a28031ce0d4d634fdac10a6c086a51131